### PR TITLE
Pavel

### PR DESCRIPTION
**Pull request by PaulGrishunin**

### CODE REVIEW

**Comment by PaulGrishunin** [comment]
List of books added. Please check if the application right receives a bocks.json.

File: client/src/BooksList/BooksList.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+//import data from './books.json';
```
Comment (ZPoreba) [inline client/src/BooksList/BooksList.js:3]: delete comment

File: client/src/Common/Layout.js [hunk 2]
```diff
@@ -56,6 +57,11 @@ class CustomLayout extends React.Component{
                 theme="light"
                 mode="horizontal"
                 style={{lineHeight: '64px'}} >
+<Menu.Item
+    key="booksList"
+    style={{float: 'right'}}>
```
Comment (ZPoreba) [inline client/src/Common/Layout.js:62]: I think it should be placed in center possision like rest of tabs so set style={{float: 'center', marginLeft: '2%'}}>

File: client/src/BooksList/BooksList.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+import './BooksList.css';
+import Rater from 'react-rater';
+import 'react-rater/lib/react-rater.css';
+
+const axios = require('axios');
+
+class BooksList extends Component {
+
+    axios.get('/books')
```
Comment (ZPoreba) [inline client/src/BooksList/BooksList.js:12]: Does it works ? It should be inside constructor or some function. I have error during compilation

**Comment by ZPoreba** [comment]
IMPORTANT!! @PaulGrishunin  I pull changes from master to your branch so now it's updated and conflicts are resolved. Please make: 1. git fetch then 2. git reset --hard origin/Pavel and then make changes

File: client/src/BooksList/BooksList.js (hunk 3 above)
Comment (ZPoreba) [reply client/src/BooksList/BooksList.js:12]: Also tests fail because of it

**Pull request merged**